src/three/DebugTilesRenderer.js

```diff
@@ -237,7 +250,8 @@ export class DebugTilesRenderer extends TilesRenderer {
 						case DEPTH: {
 
 							const val = tile.__depth / maxDepth;
-							c.material.color.setRGB( val, val, val );
+							/** map higher depth values to red, lower to green, lerping by hue */
+							c.material.color.setHSL( MathUtils.mapLinear( val, 0, 1, 0.9, 0.1 ), 1, 0.5 );
```

Comment:
> What's the motivation for scaling DEPTH and SCREEN_ERROR from red to green while keeping RELATIVE_DEPTH and GEOMETRIC_ERROR scaling from black (or near black) to white?

To be honest, I did not find RELATIVE_DEPTH or GEOMETRIC_ERROR to be useful enough that I desired the readability increase of the added contrast for those?

On the RED <-> GREEN, I don't feel strongly there, I just found it helpful to accentuate the differences even further and made it easier to explain the screen error concept to other people with a *very* visible transition

Reply:
If prefer red -> green then how do you feel about `minColor` and `maxColor` member variables that default to black and white and are used to define the min and max values of the color scale? Or a function called `getScaleColor( value )` so it can be overridden in user code. Then the scales for all debug displays are consistent and you have the flexibility to display it how you want.

Reply:
Swapped over to the min/max color and lerpHsl, works for me!